src/improvements/template/OPCMUpgradeV300.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: MIT
```

Comment:
You'll need an example task for that CI passes. This is not the actual U14 task.